### PR TITLE
update: remove dsp with v1(tekton)backend related code

### PR DESCRIPTION
**Pull request by zdtsw**
no need review at the moment, lets wait and see how these images will be if we can totally remove the logic

## Description

- images
- tekton rbac
- descriptions
depend on  https://github.com/opendatahub-io/opendatahub-operator/pull/1281

https://issues.redhat.com/browse/RHOAIENG-13906

## Merge criteria

- [ ] You have read the [contributors guide](https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/CONTRIBUTING.md).
- [ ] Commit messages are meaningful - have a clear and concise summary and detailed explanation of what was changed and why.
- [ ] Pull Request contains a description of the solution, a link to the JIRA issue, and to any dependent or related Pull Request.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: components/datasciencepipelines/datasciencepipelines.go [hunk 1]
```diff
@@ -45,22 +45,15 @@ func (d *DataSciencePipelines) Init(ctx context.Context, _ cluster.Platform) err
...
-		"IMAGESV2_ARGO_ARGOEXEC":           "RELATED_IMAGE_ODH_DATA_SCIENCE_PIPELINES_ARGO_ARGOEXEC_IMAGE",
-		"IMAGESV2_ARGO_WORKFLOWCONTROLLER": "RELATED_IMAGE_ODH_DATA_SCIENCE_PIPELINES_ARGO_WORKFLOWCONTROLLER_IMAGE",
-		"V2_DRIVER_IMAGE":                  "RELATED_IMAGE_ODH_ML_PIPELINES_DRIVER_IMAGE",
-		"V2_LAUNCHER_IMAGE":                "RELATED_IMAGE_ODH_ML_PIPELINES_LAUNCHER_IMAGE",
-		"IMAGESV2_ARGO_MLMDGRPC":           "RELATED_IMAGE_ODH_MLMD_GRPC_SERVER_IMAGE",
+		"IMAGES_DSPO":                    "RELATED_IMAGE_ODH_DATA_SCIENCE_PIPELINES_OPERATOR_CONTROLLER_IMAGE",
+		"IMAGES_APISERVER":               "RELATED_IMAGE_ODH_ML_PIPELINES_API_SERVER_V2_IMAGE",
+		"IMAGES_PERSISTENCEAGENT":        "RELATED_IMAGE_ODH_ML_PIPELINES_PERSISTENCEAGENT_V2_IMAGE",
+		"IMAGES_SCHEDULEDWORKFLOW":       "RELATED_IMAGE_ODH_ML_PIPELINES_SCHEDULEDWORKFLOW_V2_IMAGE",
+		"IMAGES_ARGOEXEC":                "RELATED_IMAGE_ODH_DATA_SCIENCE_PIPELINES_ARGO_ARGOEXEC_IMAGE",
```
Comment (HumairAK) [inline components/datasciencepipelines/datasciencepipelines.go:52]:
```suggestion
		"IMAGES_ARGO_EXEC":                "RELATED_IMAGE_ODH_DATA_SCIENCE_PIPELINES_ARGO_ARGOEXEC_IMAGE",
```

Comment (HumairAK) [reply components/datasciencepipelines/datasciencepipelines.go:52]: reference [here](https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/config/base/params.env)

**Review by HumairAK — CHANGES_REQUESTED** [review]

**Comment by HumairAK** [comment]
aside from comment above lgtm

**Comment by VaishnaviHire** [comment]
Are these not built in CPaaS https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/config/base/params.env#L10-#L11 ?

cc: @HumairAK

**Comment by zdtsw** [comment]
> Are these not built in CPaaS https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/config/base/params.env#L10-#L11 ?
> 
> cc: @HumairAK

these are borrowed from other products by pipeline

**Pull request merged**